### DISCUSSION TITLE
[WEBUI] UI short process name with argument

### DISCUSSION THREAD

**Discussion by Jerry-Ma** [nicolargo/glances#2099] [General]
Before filling this issue, please read the manual (https://glances.readthedocs.io/en/latest/) and search if the bug do not already exists in the database (https://github.com/nicolargo/glances/issues).

#### Description

It appears that for the web UI, the process argument does not display by default.

#### Feature Request

It would be great if we have either of the two:

* Add a command line option to start the web ui with long process name. (as if the "/" key was pressed)

* Show the process arguments when in short mode, which is the same behavior when using the curse ui in terminal. 

BTW, this is a great utility, and I appreciate all the hard work you guys have done to make such a great tool.
#### Versions

* Glances & psutil (glances -V):

Glances v3.1.4 with PsUtil v5.6.7

* Operating System (lsb_release -a):

Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic

By

**Comment by nicolargo** [comment]
@notFloran Can you please tweak the UI to display long process name as default behavor ?

Thanks.

**Comment by biplobsd** [comment]
I also looking for this!! Useful!

**Comment by nicolargo** [comment]
Linked to https://github.com/nicolargo/glances/issues/462

**Comment by prinsss** [comment]
Just installed glances, and surprisingly found that I can't set `--process-long-name` with `--webserver` together.

Searching around and found this issue. Though the kotkey `/` works, it would be great if there is an option to display process cmdline on web UI by default.

Anyway, thank you for the great tool!

**Comment by nicolargo** [comment]
Shortkey '/' can be used in the WebUI to switch from short to long name.
